assembl/alembic/env.py

```diff
@@ -62,15 +62,16 @@ def run_migrations_offline():
     script output.
 
     """
-    url = aws_connection_url(config.get_section('app:assembl')
+    url = connection_url(config.get_section('app:assembl')
         ) or config.get_main_option("sqlalchemy.url")
```

Comment:
Is this `or` still valid? This is the only place where you do this. Everywhere else in the code you are doing:
```
url = connection_url(settings)
if url:
    settings['sqlalchemy.url'] = url
```